### ISSUE TITLE
[OpenCL] LocalResponseNormalization unsupported

### ISSUE THREAD

**Issue by pjaaskel** [pytorch/glow#3802]
This causes at least inception_v1 to crash when targeting OpenCL (CPU backend crashes due to lack of a suitable SoftMax implementation, BTW). Is someone already working on this?

**Comment by pjaaskel** [comment]
Something wrong with this one?
